core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -190,7 +181,17 @@ case class QueueProcessorWithCircuitBreaker(
       pool.foreach(_ ! Hold(circuitBreakerSettings.closeDuration))
     }
   }
+}
+
```

Comment:
how about make it package private? Don't want to give people to false impression that they can override this.